datahub/opal.py

```diff
@@ -52,6 +52,54 @@
     "EV Status (Idle)",
 ]
 
+key_headers = [
+    "frame",
```

Comment:
I'm suggesting removing this (an "Data Frame" from `opal_header` above) because it makes more sense to use it as the index of the DataFrame (equivalent to the "name" of the Series when assigning a row)
```suggestion
```